### PR TITLE
ZENKO-3905 - redis - dashboard as code

### PR DESCRIPTION
**Pull request by lavigne958**
Generated using:
- grafanalib:0.6.2
- scalgrafanalib:1.0.0

### CODE REVIEW

**Comment by lavigne958** [comment]
Hi bert-e !

File: monitoring/redis/dashboard.json [hunk 1]
```diff
@@ -1,88 +1,70 @@
...
-  "description": "Prometheus dashboard for Redis servers",
-  "tags": [
-    "redis"
-  ],
-  "style": "dark",
-  "timezone": "browser",
+  "annotations": {
+    "list": []
+  },
+  "description": "",
```
Comment (francoisferrand) [inline monitoring/redis/dashboard.json]: missing description: "Prometheus dashboard for Redis servers"

File: monitoring/redis/dashboard.json [hunk 2]
```diff
@@ -252,1007 +162,1035 @@
...
     }
   ],
+  "refresh": "30s",
+  "rows": [],
+  "schemaVersion": 12,
+  "sharedCrosshair": false,
+  "style": "dark",
+  "tags": [
+    "redis",
+    "base-cache"
```
Comment (francoisferrand) [inline monitoring/redis/dashboard.json]: extra tag "base-cache"

**Review by francoisferrand — APPROVED** [review]

**Review by tmacro — APPROVED** [review]

**Pull request merged**